src/Steps/Release/RewriteReleaseBranches.php

```diff
@@ -253,8 +257,8 @@ protected function getTargetBranch($targetVersion, $branching, $currentBranch)
             case Branch::MAJOR:
                 return $majorBranch;
             case Branch::AUTO:
-                // Auto disables branching for unstable tags
-                if (!$targetVersion->isStable()) {
+                // Auto disables branching for alpha tags
+                if (!$targetVersion->getStability() === 'alpha') {
```

Comment:
Because new branches weren't being created for beta releases when the default "auto" branches was used.  Saves having to change the branching strategy to "minor" in the plan

Since we always start with a beta (we didn't used to) this change makes sense to me

Reply:
Ahhh okay that's what the branching strategy does. I assume this will come with a relevant change in the process doc as well.
Sounds good.

Reply:
> Ahhh okay that's what the branching strategy does

I honestly don't know what it does and whatever it does it's probably busted.  I don't even know what I'd document.

Reply:
Oh okay. I thought you'd figured it out.
No worries then, so long as it's clear from the confluence doc what to do, I'm not fussed.